docs/tutorial/mac-app-store-submission-guide.md

```diff
@@ -56,13 +110,14 @@ Then, you need to prepare three entitlements files.
   <dict>
     <key>com.apple.security.app-sandbox</key>
     <true/>
-    <key>com.apple.security.inherit</key>
-    <true/>
   </dict>
 </plist>
 ```
 
-`parent.plist`:
+#### Extra steps without `electron-osx-sign`
+
+If you are signing your app without using `electron-osx-sign`, you must ensure
+the app bundle's entitlements have at least following capacities:
```

Comment:
```suggestion
the app bundle's entitlements have at least following keys:
```

Could write "the app bundle's entitlements have at least following entitlements" here but that sounds a bit awkward. I think "keys" is a bit more correct than "capacities" though.